### DISCUSSION TITLE
BUG｜MCSL更新后与旧版的适配

### DISCUSSION THREAD

**Discussion by Casper233** [MCSLTeam/MCSL2#90] [功能建议] [answered]
- [x] 未在 [Issues](https://github.com/LxHTT/MCSL2/issues) 中看到其他相同疑问
- [x] MCSL 2 运行在受官方支持的平台
___

### 问题描述
就一个问题：如果使用了旧版本的MCSL，当更新到新版本的时候，面对新加入的目录/文件，如何保证其目录/文件合理的更新。
就如先前版本的MCSL，有/MCSL2但无/MCSL2/Logs，如何其他类似的更新能对旧版本文件进行合理的且简单的更新。

如果使用新版本的MCSL2，但是用已经生成旧的的MCSL2目录，就会出现无法打开的情况。

**Comment by AresConnor** [comment]
我觉得吧，目前用的人还是不多的，而且也是处于测试阶段，目前似乎没有正式的发行版，就这个问题而言可以考虑不向前兼容来保证代码不臃肿以及逻辑清晰

**Comment by lxhtt** [comment]
> 我觉得吧，目前用的人还是不多的，而且也是处于测试阶段，目前似乎没有正式的发行版，就这个问题而言可以考虑不向前兼容来保证代码不臃肿以及逻辑清晰

不太好。有空我就重开那个pr让Casper改改
~~**前提是我先把本地没交的先交了**~~

**Comment by lxhtt** [comment, marked as answer]
已经完善。
